pkg/services/sqlstore/dashboard.go

```diff
@@ -234,7 +234,7 @@ func (ss *SQLStore) GetFolderByTitle(orgID int64, title string) (*models.Dashboa
 
 // TODO: Remove me
```

Comment:
Can we deprecate this without linting failing?

Reply:
No, I don't want to introduce this now since linting are failing:
```
lint via golangci-lint
pkg/services/ngalert/state/manager.go:254:8: SA1019: sqlstore.GetDashboard is deprecated: Use GetDashboardCtx instead. (staticcheck)
        err = sqlstore.GetDashboard(query)
```

Reply:
Technically, we can change the severity of this rule to "warning", or to ignore SA1019 in the linter config, but that probably deserves a separate discussion.